WordPress/src/main/java/org/wordpress/android/ui/comments/CommentsListFragment.java

```diff
@@ -469,6 +469,9 @@ protected CommentList doInBackground(Void... args) {
                 int numExisting = getCommentAdapter().getCount();
                 hPost.put("offset", numExisting);
                 hPost.put("number", COMMENTS_PER_PAGE);
+            } else if (!mCanLoadMoreComments ||
```

Comment:
Is it possible to remove this part of the if condition?
If I have 200+ comments in the list, and do a PTR the app asks for 200+ comments to the server, that in some self hosted installation can lead to OOM errors. If the user launches a PTR the default number of comments should be retrieved from the server.